### DISCUSSION TITLE
[feat request] 回收站计时删除功能可能会提升用户体验

### DISCUSSION THREAD

**Discussion by shenmo7192** [linuxdeepin/developer-center#2832] [General | 常规讨论]
回收站计时器功能最早在iOS中加入，现在已经成为移动端OS的标配之一了
把删除的文件放入回收站是为了防止误删，然而回收站长期不清理可能会造成文件堆积，需要时不时清空
而回收站定时自动删除就是很好的解决方案，对于误删的文件有足够的时间还原，对于删除后不再想恢复的文件在一段时间后也可以自动删除
是很好的解决方案

从文件被放入回收站时的时间戳计时，在经过特定时间后删除

**Comment by Mile-Lile** [comment]
+1 
I would like to see this too... and default time for deleting trash should be changable...

**Comment by shenmo7192** [comment]
> +1 I would like to see this too... and default time for deleting trash should be changable...

Exactly！It would be awesome

**Comment by meiyixiang** [comment]
这是一个很好的建议，很早的时候我也提出过这样的想法，我会记录下来。

**Comment by meiyixiang** [comment]
This feature is very similar to recycling strategies in cloud services.

**Comment by meiyixiang** [comment]
Hello, we have recorded your suggestion and it will be included in the later product planning

**Reply by shenmo7192** [reply]
> Hello, we have recorded your suggestion and it will be included in the later product planning

It's half a year and nothing seems done

I noticed that deepin-album have similar funtion now.

Any plan that make dde-file-manager support it?

**Reply by shenmo7192** [reply]
> Hello, we have recorded your suggestion and it will be included in the later product planning

五百年了

**Comment by SamLukeYes** [comment]
Hmm, sounds like this: https://man.archlinux.org/man/tmpfiles.d.5.en

**Comment by shenmo7192** [comment]
每日一催

**Comment by shenmo7192** [comment]
每日一催

**Comment by shenmo7192** [comment]
五百年了

**Comment by shenmo7192** [comment]
五百年了

**Reply by meiyixiang** [reply]
别催了没人做，明年社区自己把他当差异化需求进行排期我放需求池了。

**Reply by shenmo7192** [reply]
https://bbs.deepin.org/post/258875
